spec/models/book_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
+require 'rails_helper'
+require 'faker'
+
+describe Book, type: :model do
+  it do
+    should validate_presence_of(:title)
+  end
+  it do
+    should validate_presence_of(:genre)
+  end
+  it do
+    should validate_presence_of(:author)
+  end
+  it do
+    should validate_presence_of(:image)
+  end
+  it do
+    should validate_presence_of(:year)
+  end
+  it do
+    should validate_presence_of(:publisher)
+  end
+
+  subject(:book) do
+    Book.new(
+      genre: genre, author: author, image: image, title: title,
+      publisher: publisher, year: year
+    )
+  end
+
+  let(:genre)            { Faker::Book.genre }
```

Comment:
you can move this to the spec/factories/books.rb with something like this.
![example](https://user-images.githubusercontent.com/58991676/111170491-c3abf000-8582-11eb-8937-aa86db71208b.png)

Reply:
Done